profiles/utils.py

```diff
@@ -263,6 +263,42 @@ def make_thumbnail(full_size_image, max_dimension):
     return buffer
 
 
+def make_cropped_thumbnail(full_size_image, max_dimensions):
+    """
+    Make a cropped thumbnail of the image
+
+    Args:
+        full_size_image (file):
+            A file-like object containing an image. This file will seek back to the beginning after being read.
+        max_dimensions (int, int):
+            The max width and height for the thumbnail
+    Returns:
+        BytesIO:
+            A jpeg image which is a thumbnail of full_size_image
+    """
+    pil_image = Image.open(full_size_image)
+    x, y = max_dimensions
+    if pil_image.height / pil_image.width < y / x:
```

Comment:
suggest refactoring `y / x` to an `aspect_ratio` var to make this a little cleaner